intro-HPC/ch_troubleshooting.tex

```diff
@@ -200,3 +200,48 @@ \section{Module conflicts}
 that have the same version of common dependencies. You can list all versions of
 a module with \verb|module avail|: for \verb|HMMER|, this command is \verb|module avail HMMER|.
 \fi
+
+\ifgent
+
+\section{Running software that is incompatible with host}
+
+\begin{prompt}
+%\shellcmd{module swap cluster/golett}%
+
+The following have been reloaded with a version change:
+  1) cluster/delcatty => cluster/golett
+
+%\shellcmd{module load Python/2.7.14-intel-2018a}%
+%\shellcmd{python}%
+
+Please verify that both the operating system and the processor support Intel(R) MOVBE, F16C, FMA, BMI, LZCNT and AVX2 instructions.
+
+\end{prompt}
+
+When we swap to a different cluster, the available modules change so they work for that cluster.
+That means that if the cluster and the login nodes have a different CPU architecture,
+software loaded using modules might not work.
+
+If you want to test software on the login nodes, make sure the \texttt{cluster/\defaultcluster}
+module is loaded, since the login nodes and \defaultcluster have the same CPU architecture.
+
+If modules are already loaded, and then we swap to a different cluster,
+all our modules will get reloaded. This means that all current modules will be
+unloaded and then loaded again, so they'll work on the newly loaded cluster.
```

Comment:
Mention that this may result in problems as mentioned above: when swapping to a different cluster, you can run `module purge` to unload all currently loaded modules to avoid problems (do we have a section on `purge`?)